packages/spotx/src/derives/outputPrice.ts

```diff
@@ -27,9 +27,9 @@ export function outputPrice(api: ApiInterface$Rx) {
         return api.query.cennzxSpot.coreAssetId().pipe(
             switchMap((coreAssetId: EnhancedAssetId) => {
                 if (new EnhancedAssetId(assetA).eq(coreAssetId)) {
-                    return coreToAssetOutputPrice(assetB, assetA, amountBought, api);
+                    return assetToCoreOutputPrice(assetB, assetA, amountBought, api);
```

Comment:
i don't think you need to change here. if assetA is core asset, it should be coreToAssetOutputPrice